FlagsmithClient/Classes/Internal/APIManager.swift

```diff
@@ -89,10 +89,10 @@ class APIManager : NSObject, URLSessionDataDelegate {
     
     // set the cache policy based on Flagsmith settings
     request.cachePolicy = .reloadIgnoringLocalAndRemoteCacheData
-    session.configuration.urlCache = Flagsmith.shared.cache
-    if Flagsmith.shared.useCache {
+    session.configuration.urlCache = Flagsmith.shared.cacheConfig.cache
```

Comment:
Should this line be moved into the `if` clause below?

Reply:
@matthewelwell the cache policy we set below will take care of that, regardless of what the cache is set to, so I think it makes sense to leave it as is.